Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -1184,14 +1189,16 @@ void GStreamerMediaEndpoint::onIceCandidate(guint sdpMLineIndex, gchararray cand
         if (isStopped())
             return;
 
+        String mid;
         GUniqueOutPtr<GstWebRTCSessionDescription> description;
         g_object_get(m_webrtcBin.get(), "local-description", &description.outPtr(), nullptr);
-        if (!description)
-            return;
+        if (description) {
+            if (const auto* media = gst_sdp_message_get_media(description->sdp, sdpMLineIndex))
```

Comment:
You can maybe collapse these two ifs and trust short-circuit evaluation.

Reply:
Not sure to understand what you're suggesting here :)